gradle.properties

```diff
@@ -24,3 +24,6 @@ android.nonTransitiveRClass=true
 
 org.gradle.caching=true
 org.gradle.configuration-cache=true
+
+# Roborazzi - https://github.com/takahirom/roborazzi?tab=readme-ov-file#q-how-can-i-execute-screenshot-tests-using-android-studios-run-button
+roborazzi.test.verify=true
```

Comment:
This allows us to run the screenshot tests from the ID by tapping the `run` button.